README.md

```diff
@@ -58,6 +58,12 @@ This is the source code for an API allowing management of users that have been p
     ```
     to build the images again. 
 
+    If you want to upgrade the packages in the requirements files run:
+    ```
+    > make upgrade
+    > make
```

Comment:
This is a great command.

Reply:
FYI github supports syntax highlighting for Make.

```Makefile
```

Reply:
This indeed does look amazing. Wow.